### PR TITLE
Use default static map for faster build times on `develop`.

### LINKED ISSUE #64: Fast build lane for skipping generating and requesting all static images via gatsby node
## The Problem

Build time increased very much with the new static image generation for RSV. Also every build triggers many API Calls, but we have limited request quota.

## Proposed Solution

There should be an CLI parameter or Environment variable where I can disable the generation of the static map images

## Tasks
- [ ] ?
- [ ] ...

This has some medium priority for me.

### PR DESCRIPTION
**Pull request by rush42**
Only generate static maps when `CONTEXT=porudction`. Otherwise display the [default map](https://github.com/FixMyBerlin/fixmy.rsv/blob/e3c2e3c0345fe0747b8d830d1a5f370ace450c7a/src/data/images/default_map.png).

Closes  #64.

TODO:
 - [x] switch to `CONTEXT` variable which would also exclude deploy_previews from generating static maps (see [netlify docs](https://docs.netlify.com/configure-builds/environment-variables/))

### CODE REVIEW

**Comment by ohrie** [comment]
This approach is nice, thank you!

IMHO it should be possible to enable the dynamic images to be shown on develop branch deployment on netlify, or at least, for some branches, that I can configure, so we can test it before releasing to main.

Maybe a env variable such like "FASTBUILD=True" should be used, then we can configure our [`netlify.toml`](https://docs.netlify.com/configure-builds/file-based-configuration/#build-settings) to set this variable for develop, main and if needed other branches. Since we should (or can?) not change the `CONTEXT` env variable via this, because it has [side effects](https://docs.netlify.com/site-deploys/overview/#deploy-contexts).

**Comment by ohrie** [comment]
What is the status on this merge request?

**Comment by ohrie** [comment]
@rush42 Wie ist der Stand hier? Kann das gemerged werden?

**Comment by rush42** [comment]
Hab gerade develop rein gemerged und passt alles IMO. Als nach oder vor dem merge muss dann noch die entsprechende ENV variable erstellt werden.

**Comment by ohrie** [comment]
@rush42 Du meinst in Netlify?
Das ist automatisch production, wenn wir releasen würden. Siehe https://docs.netlify.com/configure-builds/environment-variables/

Dann steht ja einem merge in develop bzw. main nichts im Wege, korrekt?

File: gatsby-node.ts [hunk 1]
```diff
@@ -105,3 +95,38 @@ export const onCreateNode = async ({
     });
   }
 };
+
+// link geometry and staticMap to metaJson nodes
+export const createSchemaCustomization = ({
+  actions: { createTypes, createFieldExtension },
+}) => {
+  if (process.env.FAST_BUILD === '1') {
```
Comment (rush42) [inline gatsby-node.ts:103]: @ohrie ich meine diese `FAST_BUILD` variable, muss noch in netlify eingetragen werden.

**Pull request merged**